docs/connector-development/config-based/tutorial/5-incremental-reads.md

```diff
@@ -11,8 +11,8 @@ First we'll update the spec block in `source_exchange_rates_tutorial/exchange_ra
 
 ```yaml
 spec: 
-  documentationUrl: https://docs.airbyte.io/integrations/sources/exchangeratesapi
-  connectionSpecification:
+  documentation_url: https://docs.airbyte.io/integrations/sources/exchangeratesapi
+  connection_specification:
```

Comment:
Remove these changes that aren't related to PR description.

Reply:
@marcosmarxm Thanks for the hint. Somehow this commit made its way into the PR. I removed it.